tokio/src/future/mod.rs

```diff
@@ -3,7 +3,7 @@
 //! Asynchronous values.
 
 mod maybe_done;
-pub(crate) use maybe_done::{maybe_done, MaybeDone};
+pub use maybe_done::{maybe_done, MaybeDone};
```

Comment:
Is this intended to be a stable API, or is it only used by the macro? If the latter, should it be `#[doc(hidden)]`?

Reply:
The `future` module is private and the type is re-exported in a `doc(hidden)` module (`tokio::macros::support`)